molgenis-core-ui/README.md

```diff
@@ -0,0 +1,83 @@
+# MOLGENIS core UI
+
+## Javascript
+We use the [frontend maven plugin](https://github.com/eirslett/frontend-maven-plugin)
+to allow maven to install node and run npm and other javascript tools.
+
+The javascript files in and other static resources src/main/javascript are packed into bundles using webpack.
+
+### Directory structure
+```
+|-- molgenis-core-ui
+    |-- README.md // this file
+    |-- gulpfile.js // javascript build actions, used to write version from pom.xml to package.json
```

Comment:
Please explain better what this does. As I understand this is used to read the POM and generate javascript with proper version number